### ISSUE TITLE
CommentsPart setting for reverse display order

### ISSUE THREAD

**Issue by BenedekFarkas** [OrchardCMS/Orchard#5168]
CommentsPartSettings should have an additional option for displaying comments in Ascending (currently wired-in) or Descending order using radio buttons.
Question is: with threaded comments enabled, descending order should be applied to the first level only, or all of them?

**Comment by dcinzona** [comment]
1st level only
Otherwise, it's hard to follow the flow of the thread.

Or make it an option?

Desc [y] [n] 
1st Level only [y] [n]

**Comment by Piedone** [comment]
Don't we want to implement system Projector queries as discussed a long time ago, make CommentsPart use such a query in the background, and let you do any fancy listing by modifying this query?
